modules/virt-hot-plugging-cpu.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-cpu_{context}"]
+
+= Hot plugging CPUs on a virtual machine
+
+You can increase or decrease the number of CPU sockets allocated to a virtual machine (VM) without having to restart the VM by using the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
+. Select the required VM to open the *VirtualMachine details* page.
+. On the *Configuration* tab, click *Edit CPU|Memory*.
+. Select the *vCPU* radio button.
+. Enter the desired number of vCPU sockets and click *Save*.
++
+If the VM is migratable, a live migration is triggered. If not, or if the changes cannot be live-updated, a `RestartRequired` condition is added to the VM.
```

Comment:
Do we generally use the term "migratable" in the docs? I would expect the more dictionary-friendly "migrate-able".